models/schema.yml

```diff
@@ -0,0 +1,67 @@
+version: 2
+
+models:    
+  - name: mp_events
+    description: "This table collects all media player events as a base table"
+    columns:
```

Comment:
I guess more column documentation will be added? And it might be worth writing your descriptions in the docs folder and then referencing the documentation written in the docs folder here

Reply:
yes, that will be a separate ticket for sure, this was a very initial draft only :)

Reply:
You will see documentation once I push the updated project.